drive/types.bal

```diff
@@ -56,41 +56,41 @@ public type About record {
     boolean canCreateDrives?;
 };
 
-# File Record Type
+# Represents File.
 @display {label: "File"}
 public type File record {
     *FileMetadata;
     *GeneratedFileMetadata;
 };
 
-# File Metadata Record Type
+# Represents file metadata.
 #
-# + name - The name of the file. This is not necessarily unique within a folder. 
+# + name - The name of the file. This is not necessarily unique within a folder.
 #          Note that for immutable items such as the top level folders of shared drives, My Drive root folder, and 
```

Comment:
Change the last sentence as follows: "Note that the name is constant for immutable items such as the top level folders of shared drives, `My Drive` root folder, and the `Application Data` folder"

Reply:
Changed as above
/Thanks